.github/README.md

```diff
@@ -1,18 +1,43 @@
 <p align="center">
-  <img alt="Files Logo" src="../src/Files.App%20(Package)/Assets/AppTiles/Release/StoreLogo.scale-400.png" width="100px" />
+  <img alt="Files Logo" src="../assets/ReadmeHero.png" />
   <h1 align="center">Files</h1>
```

Comment:
Is the extra header really needed as the names already in the image

Reply:
Good point